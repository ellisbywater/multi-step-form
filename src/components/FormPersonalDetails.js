import React, {Component} from 'react'

class FormPersonalDetails extends Component {
    continue = e => {
        e.preventDefault()
    }
    render() {
        const { values, handleChange } = this.props
        return (
            <div>
                Form Personal Details
            </div>
        )
    }
}

export default FormPersonalDetails