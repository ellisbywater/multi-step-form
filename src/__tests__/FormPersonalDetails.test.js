import React from 'react'
import { shallow } from 'enzyme'

import FormPersonalDetails from '../components/FormPersonalDetails'

describe('FormPersonalDetails', () => {
    it('should render correctly', () => {
        const component = shallow(<FormPersonalDetails />)
        expect(component).toMatchSnapshot()
    })
})
