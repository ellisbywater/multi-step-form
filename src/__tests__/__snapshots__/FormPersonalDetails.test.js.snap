// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormPersonalDetails should render correctly 1`] = `ShallowWrapper {}`;
